app/controllers/api_controller/renderer.rb

```diff
@@ -457,7 +457,7 @@ def fetch_typed_subcollection_actions(method, is_subcollection)
       return unless is_subcollection
       ctype = @req[:collection].to_sym
       sakey = "#{@req[:subcollection]}_subcollection_actions".to_sym
-      collection_config.fetch_path(ctype, sakey, method.to_sym)
+      collection_config[ctype][sakey] && collection_config[ctype][sakey][method.to_sym]
```

Comment:
I considered it, but haven't added yet.  I don't see why we couldn't patch it into Config::Options.  There is already a patch method for config, so perhaps add it there?  It's a matter of mixing in more_core_extensions/shared/nested, more or less.